vars/vars.go

```diff
@@ -11,11 +12,21 @@ var (
 	Version  = "dev" // is set during build process
 	LogDebug = os.Getenv("DEBUG") != ""
 	LogJSON  = os.Getenv("LOG_JSON") != ""
-	Genesis  = 1_606_824_023
+	Genesis  = getGenesis()
```

Comment:
you can use go-utils `cli.GetEnvInt` here, instead of creating this function here.

https://github.com/flashbots/go-utils/blob/main/cli/cli.go#L26

```
import "github.com/flashbots/go-utils/cli"

...

Genesis  = cli.GetEnvInt("GENESIS", 1_606_824_023)
```